guides/bidirectionality.md

```diff
@@ -0,0 +1,22 @@
+# Angular Material bi-directionality
+
+### How to set your application direction
+The simplest way is applying `dir` attribute on either the `html` or `body` tags.
```

Comment:
@jelbourn So, perhaps it could be a bright idea to specify that this is something to be revised. Should I open a feature request for this?